src/Sarif.TestUtilities/FileDiffingTests.cs

```diff
@@ -27,17 +105,17 @@ protected string GenerateDiffCommand(string expected, string actual)
             return String.Format(CultureInfo.InvariantCulture, "windiff \"{0}\" \"{1}\"", expected, actual);
         }
 
-        protected static bool AreEquivalentSarifLogs(string actualSarif, string expectedSarif)
+        protected static bool AreEquivalentSarifLogs<T>(string actualSarif, string expectedSarif, IContractResolver contractResolver = null)
```

Comment:
>T [](start = 53, length = 1)

Why are we exposing this?

Reply:
it is consumed external to this assembly, by sarif.unit. tests

---
In reply to: [226787959](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226787959) [](ancestors = 226787959)